events/test.py

```diff
@@ -0,0 +1,92 @@
+import unittest
+import datetime
+import app
+
+
+class TestEventsDB(unittest.TestCase):
+    def setUp(self):
+        # TODO(cmei4444): move this to a mock DB to avoid the official DB
+        # interfering with tests
+        self.example_time = datetime.datetime(2019, 6, 11, 10, 33, 1, 100000)
+        self.event_info = {'name': 'test_event',
+                           'description': 'testing!',
+                           'author': 'admin',
+                           'event_time': '7-12-2019',
+                           'created_at': self.example_time}
+        self.test_event = app.Event(self.event_info)
+        # make sure DB has no other test entries
+        app.events_coll.delete_many(self.event_info)
+
+    def test_add(self):
+        self.test_event.add_to_db()
+        queried_event = app.events_coll.find_one(self.event_info)
+        self.assertEqual(app.Event(queried_event), self.test_event)
+
+    def test_build_info(self):
```

Comment:
It's best for each test to test exactly one thing. In this case, it seems like this test is testing the time rounding behavior, so I might call it `test_time_rounding'.

Reply:
Removed since this function no longer does rounding